egret/model_library/transmission/branch.py

```diff
@@ -689,3 +809,44 @@ def declare_ineq_angle_diff_branch_lbub(model, index_set,
             m.ineq_angle_diff_branch_ub[branch_name] = \
                 pe.atan(m.vj[from_bus] / m.vr[from_bus]) \
                 - pe.atan(m.vj[to_bus] / m.vr[to_bus]) <= math.radians(branches[branch_name]['angle_diff_max'])
+
+
+def declare_ineq_p_interface_lbub(model, index_set, interfaces):
+    """
+    Create the inequality constraints for the interface limits
+    based on the power variables or expressions.
+
+    p_interface_limits should be (lower, upper) tuple
+    """
+    m = model
+    con_set = decl.declare_set('_con_ineq_p_interface_lbub',
+                               model=model, index_set=index_set)
+
+    m.ineq_pf_interface_lb = pe.Constraint(con_set)
+    m.ineq_pf_interface_ub = pe.Constraint(con_set)
+
+    slacks_pos = hasattr(m, 'pfi_slack_pos')
+    slacks_neg = hasattr(m, 'pfi_slack_neg')
+    if slacks_pos != slacks_neg:
```

Comment:
If slacks_pos and slacks_neg are both False, won't you have problems later?

Reply:
Not it in this function -- I rely on python short-circuiting in the if statements on line 839 and 848.

Other places we're iterating over InterfacesWithSlack, which will be a no-op if these are both False.